js/jquery.validation.js

```diff
@@ -143,32 +181,38 @@
                 var ajax_method;
                 if ( method === 'POST' || method === 'GET' ) {
                     ajax_method = method;
+
                 } else {
                     // Emulate HTTP method
-                   ajax_method = 'POST';
-                    var found = false;
-                    $.each(data, function(index, item) {
-                        if ( item.name === '_method' ) {
-                            item.value = method;
-                            found = true;
-                            return;
-                        }
-                    });
+                    ajax_method = 'POST';
+
+                    if ( isFormData ) {
+                        data.set('_method',method);
```

Comment:
for tlc-www at least we always set the form method attribute to POST and add a hidden _method form variable to perform overloading so agree this would not break existing tlc-www functionality.

Might cause issues if developer set the form method attribute to be PUT though ... not aware of us doing that at the moment though